### PR TITLE
Add support for PHP 8

### PR DESCRIPTION
**Pull request by joskfg**
Allow PHP 8 versions. Related with #39

### CODE REVIEW

**Comment by BramG** [comment]
Can someone commit this? It works as expected.

**Review by vuphuong87 — APPROVED** [review]

**Comment by vuphuong87** [comment]
Guess we can merge this one @gmponos ?
Thanks

**Comment by coccoinomane** [comment]
The changes work for me too 👍
@gmponos would you mind accepting the pull request?

**Comment by kprovorov** [comment]
Hi all! Any updates? :)

File: composer.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
         }
     },
     "require": {
-        "php": "^7.2",
+        "php": "^7.2 || ^8",
```
Comment (GrahamCampbell) [inline composer.json]: `^7.2 || ^8.0` is the convention here

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

**Comment by gmponos** [comment]
Thank you

**Comment by vuphuong87** [comment]
Could you release a new version for the recent merges? Thanks @gmponos

**Comment by kingeke** [comment]
Hey @gmponos pls we're waiting for the new version, much thanks.

**Pull request merged**